### DISCUSSION TITLE
Edge shows blank when starting th App

### DISCUSSION THREAD

**Discussion by JoviLiu0919** [lllyasviel/Fooocus#3170] [Q&A] [answered]
Hi there, 

I got a problem, I cannot succefully launch the app of Fooocus, and got blank content on Edge.
![image](https://github.com/lllyasviel/Fooocus/assets/173336992/90046326-9beb-4ab7-87c6-b57da325b47e)
![image](https://github.com/lllyasviel/Fooocus/assets/173336992/fc58031f-0484-4198-b422-03faa894d53d)

How to check and reslove?
Thanks for your help.

**Comment by ILikeToasters** [comment, marked as answer]
You could try a different browser but I think your issue might be something on your system like security software or something else blocking it because Edge should work.

**Reply by JoviLiu0919** [reply]
ok, let me try, thanks!
